inc/switchtec/switchtec.h

```diff
@@ -482,7 +513,15 @@ int switchtec_fw_cfg_info(struct switchtec_dev *dev,
 			  struct switchtec_fw_image_info *inact_cfg,
 			  struct switchtec_fw_image_info *mult_cfg,
 			  int *nr_mult);
-int switchtec_fw_img_write_hdr(int fd, struct switchtec_fw_footer *ftr,
+int switchtec_bl2_img_info(struct switchtec_dev *dev,
```

Comment:
suggests rename to switchtec_fw_bl2_info to align with other similar API.

Reply:
yes, modify it.